docs/MAAP_USAGE.md

```diff
@@ -82,43 +84,53 @@ To run a GEDI subsetting DPS job, you must supply the following inputs:
   quality_flag == 1 and `geolocation/sensitivity_a2` > 0.95
   ```
 
-- `limit` (_optional_; default: 1_000): Maximum number of GEDI granule data
-  files to download from the CMR, among those that intersect the specified AOI's
-  bounding box, and fall within the specified temporal range (if supplied).
-
-  _Changed in version 0.6.0_: The default value was reduced from 10000 to 1000.
-  The AOI for most subsetting operations are likely to incur a request for well
-  under 1000 granules for downloading, so a larger default value might only lead
-  to longer CMR query times.
+- `limit` (_optional_; default: 100000): Maximum number of GEDI granule data
+  files to subset, among those that intersect the specified AOI's bounding box,
+  and fall within the specified temporal range (if supplied).  If there are more
+  granules within the spatio-temporal range, only the first `limit` number of
+  granules obtained from the corresponding CMR search are used.
+
+  > _Changed in version 0.6.0_: The default value was reduced from 10000 to 1000.
+  > The AOI for most subsetting operations are likely to incur a request for well
+  > under 1000 granules for downloading, so a larger default value might only lead
+  > to longer CMR query times.
+
+  > _Changed in version 0.8.0_: The default value was increased from 1000 to
+  > 100000 to avoid confusion in cases where a user does _not_ specify a limit,
```

Comment:
I think the problem here is now a user is unlikely to ever set a limit. How much extra time is required to run such a large job if the bbox is global? Somehow I think we need to find a better way to inform users of the limit but still keep it lower.

Reply:
See my other comment about the problem with setting an arbitrarily low limit.

Reply:
I'm going to suggest we poll/ask the UWG.

Reply:
This was the source for this enhancement request. It stemmed from precisely this problem I described above.

Reply:
See https://esa-nasa-maap.slack.com/archives/C05J4SR9021/p1713901061380449

Reply:
oh I remember the issue with the users, but that's only the opinion of 1 user. I think a discussion needs to happen at the UWG level about what the best approach is.

Reply:
Neha Hunka, Paromita Basak, Abigail Barenblitt, and Veronika Leitold have all voted in favor of 100000 as the default.

Reply:
ok, lets go with that for now then. We can revisit the question of too large of outputs later.